python_modules/dagster/dagster/core/definitions/sensor_definition.py

```diff
@@ -141,14 +141,18 @@ def repository_name(self) -> Optional[str]:
 SensorExecutionContext = SensorEvaluationContext
 
 RawSensorEvaluationFunctionReturn = Union[
-    Iterator[Union[SkipReason, RunRequest]], List[RunRequest], SkipReason, RunRequest
+    Iterator[Union[SkipReason, RunRequest, PipelineRunReaction]],
```

Comment:
ok - this is from a mypy complaint because there is a list made later that gets the PipelineRunReactions that are returned by the sensor (if any). other option if to just ignore the mypy warning for that line